assets/js/block-variation.js

```diff
@@ -0,0 +1,40 @@
+
+const { __ } = wp.i18n;
+
+wp.blocks.registerBlockVariation( 'core/query', {
+	name: 'twentytwentyfour/featured-image-query',
+	title: __( 'Featured Images' ),
+	description: __( 'List of posts with featured images' ),
+	attributes: {
+		namespace: 'twentytwentyfour/featured-image-query',
+		query: {
+			perPage: 6,
+			postType: 'post',
+			order: 'desc',
+			orderBy: 'date',
+			inherit: false,
+			hasFeaturedImage: true,
```

Comment:
@ryanwelcher Do you have time look at this query loop variation? I can't get this custom query parameter (hasFeaturedImage) to work.
Yes I read the docs, Justin's article, your plugin code, _and_ watched the hallway hangout video 😆

Reply:
Sure thing! I'll have a look when I am back in the office on Monday